versions/3.1.1.md

```diff
@@ -2790,8 +2788,15 @@ The OpenAPI Specification allows combining and extending model definitions using
 `allOf` takes an array of object definitions that are validated *independently* but together compose a single object.
 
 While composition offers model extensibility, it does not imply a hierarchy between the models.
-To support polymorphism, the OpenAPI Specification adds the `discriminator` keyword.
+To support polymorphism, the OpenAPI Specification adds the `discriminator` field.
```

Comment:
I cross-checked our predominant use of "keyword" and "field":
- "keyword" if it is defined by JSON Schema
- "field" if it is defined by OAS (in one of the "Fixed Fields" sections)